tests/test_x86.c

```diff
@@ -50,7 +50,19 @@ static void print_insn_detail(csh ud, cs_mode mode, cs_insn *ins)
 
 	printf("\taddr_size: %u\n", x86->addr_size);
 	printf("\tmodrm: 0x%x\n", x86->modrm);
+	if (x86->encoding.modrm_offset != 0) {
+	    printf("\tmodrm_offset: 0x%x\n", x86->encoding.modrm_offset);
```

Comment:
we are using tabs for indentation in C code, but looks like this is using spaces?